examples/heat-source-mpi.py

```diff
@@ -118,6 +119,17 @@ def rhs(t, u):
     t_final = 0.01
     istep = 0
 
+    # initialize Leap integrator if using one
+    if use_leap:
+        timestepper = RK4MethodBuilder("u")
+        code = timestepper.generate()
+        from dagrt.codegen import PythonCodeGenerator
+        codegen = PythonCodeGenerator(class_name="Method")
+        interp = codegen.get_class(code)(function_map={
+            "<func>" + "u": rhs,
+            })
+        interp.set_up(t_start=t, dt_start=dt, context={"u": u})
+
```

Comment:
This too gets pretty repetitive. Package it up somewhere?

Reply:
This is done in 0f65f55, though I'm not at all sure if it's where it should be, or if it's what you want it to look like.